package.json

```diff
@@ -183,7 +183,7 @@
     "@testing-library/dom": "7.30.3",
     "@testing-library/jest-dom": "5.11.9",
     "@testing-library/react": "11.2.5",
-    "@testing-library/user-event": "12.8.0",
+    "@testing-library/user-event": "13.0.0",
```

Comment:
We may as well use the latest version of `user-event` (currently 13.2.1) when we're making this upgrade.